### ISSUE TITLE
esp32_env: What are the plans for ADC, DAC and PWM

### ISSUE THREAD

**Issue by gschorcht** [RIOT-OS/RobotFW-tests#57]
As the title says, what are the plans for ADC, DAC and PWM?

ADC, DAC and PWM pins are already listed in the environment configuration and also defined as PHiLIP ports. But it is not clear to me on how to map the the GPIO definition in the environment definition to an ADC/DAC line or a PWM channel.

Dependent on the plans for ADC, DAC and PWM, the GPIOs in `esp32-wroom-32.env` should be different. For example, using GPIOs 25, 26, 27 for GPIO tests would prevent to test a DAC pin.

**Comment by MrKevinWeiss** [comment]
Yup, we should be supporting it for the next release.  I will make sure the pins are connected to the appropriate periphs.
